src/chocolatey.resources/helpers/functions/Get-WebFile.ps1

```diff
@@ -177,17 +177,16 @@ param(
     $req.UserAgent = $userAgent
   }
 
-  if ($options.Headers.Count -gt 0) {
+  if($options.Headers.Count -gt 0){
     Write-Debug "Setting custom headers"
-    foreach ($item in $options.Headers.GetEnumerator()) {
-      $uri = (new-object system.uri $url)

```

Comment:
@steviecoaster looks like you cut this one out by accident. It's used when setting the cookies so you'll probably need that. 🙂

Reply:
Fixed